modules/contact/test/tests/simple_contact/two_block_compress/tests

```diff
@@ -11,6 +11,17 @@
     min_ad_size = 100
     abs_zero = 1.0e-8
   []
+  [frictionless-compress-2d_pg]
+    type = 'CSVDiff'
+    input = 'two_equal_blocks_compress_2d_pg.i'
+    csvdiff = 'two_equal_blocks_compress_2d_pg_out.csv'
+    mesh_mode = 'REPLICATED'
```

Comment:
Please put a comment as to why this test can only be run with replicated mesh

Reply:
It should not need to, if the non-pg version works. Let me try removing it